docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1096,3 +1098,55 @@ function su_humsci_profile_update_9718() {
   // Uninstalls modules after removing configs to prevent updb errors.
   \Drupal::service('module_installer')->uninstall(['hs_webform', 'webform_ui', 'webform']);
 }
+
+/**
+ * Add new logout block to existing sites.
+ */
+function su_humsci_profile_update_9719() {
```

Comment:
Thinking about this a bit more, this update hook is not needed. We force import configs prefixed with `block.block.humsci_colorful_` and `block.block.humsci_traditional_`. So any changes to those configs would be reverted during config import anyways.

ref: https://github.com/SU-HSDO/suhumsci/blob/develop/config/default/config_ignore.settings.yml#L10